src/SLCore.Listeners.UnitTests/Implementation/AnalysisListenerTests.cs

```diff
@@ -203,7 +203,7 @@ public void RaiseIssues_HasIssuesIntermediate_DoNotPublishIssues()
 
         var analysisService = Substitute.For<IAnalysisService>();
 
-        var testSubject = CreateTestSubject(analysisService);
+        var testSubject = CreateTestSubject(analysisService: analysisService);
```

Comment:
why is this in the diff?

Reply:
rebase issue

Reply:
Yeah but this change is already in the feature branch. How is it in the diff?

Reply:
Also I suggest rebasing instead of merging. My pr doesn't have this issue

Reply:
I saw that happen before I guess it's a git weirdness.

Reply:
I did rebase but it created a merge commit for some reason.

Reply:
Because you didn't force push

Reply:
Or Idk, the merge commit is weird